staging/src/k8s.io/apiserver/pkg/admission/plugins.go

```diff
@@ -81,7 +81,7 @@ func (ps *Plugins) Register(name string, plugin Factory) {
 		ps.registry = map[string]Factory{}
 	}
 
-	klog.V(1).Infof("Registered admission plugin %q", name)
+	klog.V(1).InfoS("Registered admission", "plugin", name)
```

Comment:
```suggestion
	klog.V(1).InfoS("Registered admission plugin", "plugin", name)
```
I think repetition here make sense as "Registered admission" itself doesn't convey what happen.